perf/perfevents_24_7.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+#
+# Copyright: 2016 IBM
+# Author: Athira Rajeev<atrajeev@linux.vnet.ibm.com>
+
+import os
+import sys
+import re
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+
+
+class test_eliminate_domain_suffix(Test):
+    """
+    This tests domain name suffix in event names
+    """
+
+    def setUp(self):
+        """
+        Check if kernel version has the supported feature
+        This feature is enabled from 3.10.0.419
+        """
+
+        # Compare kernel version
+        kernel_ver = os.uname()[2]
+        split_value = kernel_ver.split('.')
+        kernel_Version = ".".join(split_value[0:3]).replace("-", ".")
+        kernel_to_compare = "3.10.0.419"
+
+        result_comparison = 0
+
+        for cur_ker, supported_ker in zip(kernel_Version.split(".")[:4],
+                                          kernel_to_compare.split(".")[:4]):
+            if cur_ker > supported_ker:
+                result_comparison = 1
+
+        if result_comparison != 1:
+            raise self.log.warn(
+                "This test is not supported on this kernel version")
+
+    def test(self):
+        nfail = 0
+
+        # Check if this is a guest
+        # 24x7 is not suported on guest
+        cpu_output = process.run("cat /proc/cpuinfo")
+        if "emulated by" in cpu_output.stdout:
+            self.skipTest("This test is not supported on guest")
+
+        # Check if 24x7 is present
+        if os.path.exists("/sys/bus/event_source/devices/hv_24x7"):
```

Comment:
`/sys/bus/event_source/devices/hv_24x7` was used 4 times, can you please create a variable.